source/mslite-metadata.json

```diff
@@ -0,0 +1,2017 @@
+        { "name": "newWidth", "type": "long", "default": 0 },
+        { "name": "alignCorners", "type": "bool", "default": false },
+        { "name": "preserveAspectRatio", "type": "bool", "default": false }
+      ]
+    }
+  },
+  {
+    "name": "Return"
+  },
+  {
+    "name": "Reverse",
+    "schema": {
+      "inputs": [
+        { "name": "input" }
+      ],
+      "outputs": [
+        { "name": "output" }
+      ],
+      "attributes": [
+        { "name": "axis", "type": "[int]" }
+      ]
+    }
+  },
+  {
+    "name": "ReverseSequence",
+    "schema": {
+      "inputs": [
+        { "name": "input" },
+        { "name": "seq_lengths" }
+      ],
+      "outputs": [
+        { "name": "output"}
+      ],
+      "attributes": [
+        { "name": "seqAxis", "type": "int", "default": 0 },
+        { "name": "batchAxis", "type": "int", "default": 0 }
+      ]
+    }
+  },
+  {
+    "name": "Rfft",
+    "schema": {
+      "attributes": [
+        { "name": "fftLength", "type": "int", "default": 0 }
+      ]
+    }
+  },
+  {
+    "name": "ROIPooling",
+    "schema": {
+      "inputs": [
+        { "name": "input" }
+      ],
+      "outputs": [
+        { "name": "output" }
+      ],
+      "attributes": [
+        { "name": "pooledH", "type": "int", "default": 0 },
+        { "name": "pooledW", "type": "int", "default": 0 },
+        { "name": "scale", "type": "float", "default": 0.0 }
+      ]
+    }
+  },
+  {
+    "name": "Round",
+    "schema": {
+      "inputs": [
+        { "name": "x" }
+      ],
+      "outputs": [
+        { "name": "y" }
+      ]
+    }
+  },
+  {
+    "name": "Rsqrt",
+    "schema": {
+      "inputs": [
+        { "name": "x" }
+      ],
+      "outputs": [
+        { "name": "y" }
+      ]
+    }
+  },
+  {
+    "name": "Scale",
+    "schema": {
+      "outputs": [
+        { "name": "output" }
+      ],
+      "attributes": [
+        { "name": "activationType", "type": "ActivationType" },
+        { "name": "axis", "type": "int", "default": 0 }
+      ]
+    }
+  },
+  {
+    "name": "Scatter",
+    "schema": {
+      "outputs": [
+        { "name": "output" }
+      ]
+    }
+  },
+  {
+    "name": "ScatterND",
+    "schema": {
+      "outputs": [
+        { "name": "output" }
+      ]
+    }
+  },
+  {
+    "name": "Select",
+    "schema": {
+      "outputs": [
+        { "name": "output" }
+      ]
+    }
+  },
+  {
+    "name": "Shape",
+    "schema": {
+      "inputs": [
+        { "name": "input"}
+      ],
+      "outputs": [
+        { "name": "output" }
+      ]
+    }
+  },
+  {
+    "name": "Sin",
+    "schema": {
+      "inputs": [
+        { "name": "x" }
+      ],
+      "outputs": [
+        { "name": "y" }
+      ]
+    }
+  },
+  {
+    "name": "SkipGram",
+    "schema": {
+      "inputs": [
+        { "name": "input"}
+      ],
+      "outputs": [
+        { "name": "output" }
+      ],
+      "attributes": [
+        { "name": "includeAllGrams", "type": "bool", "default": false },
```

Comment:
Type names need to be consistent for all `-metadata.json` files. For example, use `boolean` instead of `bool`.